--- a/Clients/Makefile	2020-03-03 02:19:22.000000000 +0300
+++ b/Clients/Makefile	2020-03-03 02:11:20.000000000 +0300
@@ -31,10 +31,13 @@
 
 # On OS X the dns_sd library functions are included in libSystem, which is implicitly linked with every executable
 # If /usr/lib/libSystem.dylib exists, then we're on OS X, so we don't need also to link the "dns_sd" shared library
+ifneq ($(os),linux-uclibc)
 ifneq "$(wildcard /usr/lib/libSystem.dylib)" ""
 TARGETS = build/dns-sd build/dns-sd64
 LIBS =
-else
+endif
+endif
+ifeq "$(TARGETS)" ""
 TARGETS = build/dns-sd
 LIBS ?= -L../mDNSPosix/build/prod/ -ldns_sd
 endif
